pkg/util/log/log.go

```diff
@@ -558,14 +569,24 @@ func TracefStackDepth(depth int, format string, params ...interface{}) {
 
 // TracecStackDepth logs at the trace level with context and the current stack depth plus the additional given one
 func TracecStackDepth(message string, depth int, context ...interface{}) {
-	logContext(seelog.TraceLvl, func() { Tracec(message, context...) }, Logger.trace, message, depth, context...)
+	logFormatContext(seelog.TraceLvl, func() { Tracec(message, context...) }, Logger.trace, message, nil, depth, context...)
 }
 
 // Tracec logs at the trace level with context
 func Tracec(message string, context ...interface{}) {
 	TracecStackDepth(message, 1, context...)
 }
 
+// TracefcStackDepth logs at the trace level with context and the current stack depth plus the additional given one
+func TracefcStackDepth(depth int, context []interface{}, message string, params ...interface{}) {
+	logFormatContext(seelog.TraceLvl, func() { Tracefc(context, message, params...) }, Logger.trace, message, params, depth, context...)
+}
+
+// Tracefc logs at the trace level with context
+func Tracefc(context []interface{}, message string, params ...interface{}) {
+	TracefcStackDepth(1, context, message, params...)
+}
+
```

Comment:
Those need to be added to the log component too. We want the component and the package to have the same API. This will allow us to use the component instead of the package at some point.

Reply:
I reverted `DebugContextf` changes to keep this PR small, will work on a followup PR